content/docker-hub/download-rate-limit.md

```diff
@@ -1,58 +1,97 @@
+| `pat_name`           | Name of the personal access token that was used for authentication with Docker CLI.                                                                                                                                        | Personal access tokens are usually used to authenticate automated tools (Docker Desktop, CI/CD tools, etc.). This is useful for identifying which automated system issued the pull. |
+| `ips`                | The IP address that was used to pull the image. This field is aggregated, so more than one IP address may appear, representing all the IPs used to pull an image within the same date and hour.                            | This helps you understand the origin of the data transfer, which is useful for diagnosing and identifying patterns in automated or manual pulls.                              |
+| `repository_privacy` | The privacy state of the image repository that was pulled. This can either be `public` or `private`.                                                                                                                       | This distinguishes between public and private repositories to identify which data transfer threshold the pull impacts.                                                        |
+| `tag`                | The tag for the image. The tag is only available if the pull request included a tag.                                                                                                                                       | This helps in identifying the image. Tags are often used to identify specific versions or variants of an image.                                                               |
+| `digest`             | The unique image digest for the image.                                                                                                                                                                                     | This helps in identifying the image.                                                                                                                                          |
+| `version_checks`     | The number of version checks accumulated for the date and hour of each image repository. Depending on the client, a pull request can do a version check to verify the existence of an image or tag without downloading it. | This helps identify the frequency of version checks, which you can use to analyze usage trends and potential unexpected behaviors.                                          |
+| `pulls`              | The number of pulls (HEAD & GET requests) accumulated for the date and hour of each image repository.                                                                                                                      | This helps identify the frequency of repository pulls, which you can use to analyze usage trends and potential unexpected behaviors.                                          |
```

Comment:
is 1 pull == 1 `docker pull`? The "HEAD & GET requests" bit made me wonder if e.g. pulling an image with a lot of layers would result in many requests, and therefore a higher pull count than images with fewer layers? Maybe worth clarifying.

edit: I see that pull requests are defined in the next section on rate limits. Maybe it's good enough, or if we wanted to, we could probably have a section that defines more clearly what constitutes a pull, in terms of requests, that's separate from the rate limit section. Your call!